ddl/ddl_api.go

```diff
@@ -2272,6 +2281,13 @@ func (d *ddl) AlterTable(ctx sessionctx.Context, ident ast.Ident, specs []*ast.A
 	}
 
 	if len(validSpecs) > 1 {
+		if isDropIndexes(validSpecs) {
```

Comment:
What if `alter table add column, drop index`?

Reply:
This requirement does not match the title（dropping multi-indexes in one statement）😂 @wjhuang2016